pallets/loans/src/mock.rs

```diff
@@ -81,8 +73,12 @@ impl frame_system::Config for Runtime {
     type OnSetCode = ();
 }
 
+pub type AccountId = u128;
 pub type BlockNumber = u64;
 
+pub const ALICE: AccountId = 1;
```

Comment:
There is no need to exact a number to Alice in my opinion.

Reply:
I agree too, as zhou yang is developing a unit test about pallet loans, I think we can leave it for now.